lib/gitlib_test.sh

```diff
@@ -156,4 +158,28 @@ TEST_get_team_members() {
     'get_team_members can handle invalid responses and reports no members'
 }
 
+TEST_pending_prs() {
+  echo "Testing gitlab::pendig_prs"
+  echo
+
+  local branch='release-1.12'
```

Comment:
```suggestion
  local branch='release-1.15'
```

Reply:
not relevant for the test

Reply:
Not relevant for the test, but from the human perspective, it gives a bit of a signal on when we touched that piece of code w/o having to search history.